lib/tasks/evaluateme_tasks.rake

```diff
@@ -1,4 +1,85 @@
-# desc "Explaining what the task does"
-# task :evaluateme do
-#   # Task goes here
-# end
+# Rake tasks to set the routes, assets, install migrations and create initializer file.
+
+desc "Setting all the required configuration for the plugin in the application"
+
+namespace :evaluateme do
+
+  task :install do
+    Rake::Task["evaluateme:set_routes"].invoke
+    Rake::Task["evaluateme:set_assets_path"].invoke
+    Rake::Task["evaluateme:install_migrations"].invoke
+    Rake::Task["evaluateme:initializer_file"].invoke
+  end
+
+  # Set routes path for the plugin
+  task :set_routes do
+    target  = "config/routes.rb"
+    set_routes_plugin = "mount Evaluateme::Engine => '/evaluateme'"
+    routes_defined = false
+  
+    # Check the routes line of plugin already exists?
+    File.open(target, 'r+') do |f|
+      f.each do |line|
+        routes_defined = true if line.include? set_routes_plugin
+      end
+
+      # Write the routes path if not already set.
+      unless routes_defined
+        f.rewind
+        f.each do |line|
+          break  if line.include? "Rails.application.routes.draw do"
+        end
+        pos = f.pos                     # save your position in the file
```

Comment:
Mind the tabs we are using for comments. Same number of tabs will make comments easy to read and will beautify the code.

E.g.:
```
blah blah                                               # save your position in the file
blah                                                       # save the rest of the file
blah blah blah                                       # go back to the old position
```

instead of

```
blah blah                                                     # save your position in the file
blah                                                       # save the rest of the file
blah blah blah                                                   # go back to the old position
```